Makefile

```
@@ -159,6 +163,9 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
+swag: swag-install
```

I think this one also requires `make build-local`.
```suggestion
swag: swag-install build-local
```

---

It does not require a fresh build.

---

I see.